fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/api/attachment/v1/AttachmentType.java

```diff
@@ -39,13 +43,15 @@
  * immutable types. More generally, different attachments <i>must not</i> share mutable state, and it is <i>strongly advised</i>
  * for attachments not to hold internal references to their target. See the following note on entity targets.</p>
  *
- * <p>Note on {@link Entity} targets: in several instances, the name needs to copy data from one {@link Entity} to another.
- * These are player respawning, mob conversion, return from the End and cross-world entity teleportation. By default,
- * attachments are simply copied wholesale, up to {@link #copyOnDeath()}. Since one entity instance is discarded,
- * an attachment that keeps a reference to an {@link Entity} instance can and will break unexpectedly. If,
- * for whatever reason, keeping to reference to the target entity is absolutely necessary, be sure to use
- * {@link ServerPlayerEvents#COPY_FROM}, {@link ServerEntityWorldChangeEvents#AFTER_ENTITY_CHANGE_WORLD}
- * and a mixin into {@link MobEntity#convertTo(EntityType, boolean)} to implement custom copying logic.</p>
+ * <p>Note on {@link Entity} and {@link Chunk} targets: in several instances, the game needs to copy data from one instance to another.
+ * These are player respawning, mob conversion, return from the End, cross-world entity teleportation, and conversion of a {@link ProtoChunk} to
+ * {@link WorldChunk}. By default, attachments are simply copied wholesale, up to {@link #copyOnDeath()}. Since one instance is discarded,
+ * an attachment that keeps a reference to an {@link Entity} or {@link ProtoChunk} instance can and will break unexpectedly. If,
+ * for whatever reason, keeping a reference to the target is absolutely necessary, be sure to implement custom copying logic.
+ * For {@link Entity} targets, use {@link ServerPlayerEvents#COPY_FROM}, {@link ServerEntityWorldChangeEvents#AFTER_ENTITY_CHANGE_WORLD},
+ * and a mixin into {@link MobEntity#convertTo(EntityType, boolean)}. For {@link Chunk} targets, mixin into
```

Comment:
This is a bit unrelated to the PR, but might as well tackle it here. The part about the mixin should instead point to `LivingEntityEvents#MOB_CONVERSION`, I must've missed this when updating the original PR.